apps/www/scripts/build-registry.mts

```diff
@@ -41,9 +41,14 @@ export const Index: Record<string, any> = {
 
     // Build style index.
     for (const item of registry) {
-      const resolveFiles = item.files.map(
-        (file) => `registry/${style.name}/${file}`
-      )
+      const resolveFiles = item.files.map((file) => {
+        switch (file.split(".")[1]) {
+          case "stories":
+            return `registry/${file}`
+          default:
+            return `registry/${style.name}/${file}`
+        }
+      })
```

Comment:
My thinking is that a switch statement here will be more extendable in the future incase there are other supporting file types independent of the style (thinking maybe the hooks?)